genfragments/ThirteenTeV/Higgs/HY/ResonanceDecayFilter_example_HY_XToYH_HTo2G_YTo2WTo2Q1L1Nu.py

```diff
@@ -23,23 +23,25 @@
     PythiaParameters = cms.PSet(
         pythia8CommonSettingsBlock,
         pythia8CP5SettingsBlock,
-        pythia8PSweightsSettingsBlock,      
+        pythia8PSweightsSettingsBlock,
         processParameters = cms.vstring(
-            '25:onMode = off', # Turn off all H decays 
-            '25:oneChannel = 1 1 100 22 22', # H->gamma gamma  
+            '25:onMode = off', # Turn off all H decays
+            '25:oneChannel = 1 1 100 22 22', # H->gamma gamma
+            '25:onIfMatch = 15 -15',
```

Comment:
Hello! @acarvalh 

This line should be `25:onIfMatch = 22 22` (I think you know but probably a typo / incorrect copy paste)

Reply:
fixed, along with other copy/paste typo

Reply:
sorry, i'm a bit confused with the update. I guess @atishelmanch was suggesting change "25:oneChannel = 1 1 100 22 22" to "25:onIfMatch = 22 22", right? And "oneChannel" in other fragments should be checked also if it's really wanted.

Reply:
Hi @menglu21 , no, he just linked the wrong line (checked offline). He just wanted me to fix the line that defines tau decays (line 30).

Due this comment I checked again all the cards and found only one other similar typo typo, that was corrected in this same commit

Reply:
Hello @menglu21 - I confirm what @acarvalh has said.